### PR TITLE
Readme pm updates

### PR DESCRIPTION
**Pull request by rosiegrant**
a few minor updates to the read me around:
- allowEmptySearch attribute for the search bar
- analytics events

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2089,6 +2092,22 @@ You may add additional, custom analytic events to templates using certain data a
 </button>
 ```
 
+## Built-In Analytics Events For CTAs
```
Comment (creotutar) [inline README.md:2085]: Is this to be more explicit about the event types? https://github.com/yext/answers#calls-to-action
(They're listed as comments in the Calls to Action section)

Comment (rosiegrant) [reply README.md:2085]: yes! especially for someone referencing this who is building things on jambo. I'm open to removing it in the Calls to Action section if you think it's redundant?

Comment (creotutar) [reply README.md:2085]: ah nice yeah. can we remove and refer to this section in the Calls to Action section? is harder to remember to change in two places if this list ever gets updated

**Review by creotutar — APPROVED** [review]
👍 👍 👍 LGTM

**Pull request merged**